include/swift/Serialization/ModuleFormat.h

```diff
@@ -1496,6 +1494,17 @@ namespace decls_block {
     BCFixed<1> // specialization kind
   >;
 
+  // SWIFT_ENABLE_TENSORFLOW
+  using DifferentiableDeclAttrLayout = BCRecordLayout<
+    Differentiable_DECL_ATTR,
+    BCFixed<1>, // Differentiation mode ('forward' or 'reverse').
+    IdentifierIDField, // Primal name.
+    DeclIDField, // Primal function declaration.
+    IdentifierIDField, // Adjoint name.
+    DeclIDField, // Adjoint function declaration.
+    BCArray<BCFixed<32>> // Differentiation parameters.
+  >;
+
```

Comment:
VERSION_MINOR in this file needs to be bumped.

Reply:
Thanks! Done in https://github.com/apple/swift/pull/17155/commits/d7ea8bc368c53b8e5fe9600306bff33123516eb1.